patches/better-help-settings/matrix-react-sdk+3.63.0.patch

```diff
@@ -1,37 +1,72 @@
+diff --git a/node_modules/matrix-react-sdk/src/components/views/elements/AccessibleButton.tsx b/node_modules/matrix-react-sdk/src/components/views/elements/AccessibleButton.tsx
+index 3f40961..aca97f2 100644
+--- a/node_modules/matrix-react-sdk/src/components/views/elements/AccessibleButton.tsx
++++ b/node_modules/matrix-react-sdk/src/components/views/elements/AccessibleButton.tsx
+@@ -75,7 +75,7 @@ type IProps<T extends keyof JSX.IntrinsicElements> = DynamicHtmlElementProps<T>
+     disabled?: boolean;
+     className?: string;
+     triggerOnMouseDown?: boolean;
+-    onClick: ((e: ButtonEvent) => void | Promise<void>) | null;
++    onClick?: ((e: ButtonEvent) => void | Promise<void>) | null;
```

Comment:
`OnClick?` : 
In element, we have : 
https://github.com/matrix-org/matrix-react-sdk/blob/79033eb03408a58c714c47162fe9923ed9247380/src/components/views/elements/AccessibleButton.tsx#L78

Why do we need to change this?